gcloud-java-examples/README.md

```diff
@@ -39,6 +39,17 @@ To run examples from your command line:
 
 4. Run an example using Maven from command line.
 
+  Here's an example run of `BigQueryExample`.
+
+  Before running the example, go to the [Google Developers Console][developers-console] to ensure that BigQuery API is enabled.
```

Comment:
Yes, it would be good to mention the option of uploading the file using the GCS [web browser](https://pantheon.corp.google.com/storage/browser).

As for the direction for the CSV I wonder if we could actually include a sample file?